killing-floor-2config.json

```diff
@@ -69,6 +30,18 @@
             "KFGameContent.KFGameInfo_Objective": "Objective"
         }
     },
+    {
+        "DisplayName": "Mutators",
```

Comment:
Are there other things besides mutators that this could be used for? As in, should this just be a "Customer Arguments" input or are mutators all a person would ever add?

Reply:
Hmm. Technically yeah you can use the field to override any paramter after maxplayer(The fact you can do that from server also) so yeah "customer arguments" would be the best for it. For executing "bad" stuff im not sure the field is positioned in the middle of the command line and the server is already started. I will dig deeper when i have more time how i can do the best way and i will come back.
Kinda rare when people modding KF2 server so not really important part but its always good to have.